relays/substrate/src/headers_target.rs

```diff
@@ -91,11 +91,11 @@ where
 		let decoded_response: Vec<(P::Number, P::Hash)> =
 			Decode::decode(&mut &encoded_response.0[..]).map_err(SubstrateError::ResponseParseFailed)?;
 
-		let best_header = decoded_response.last().ok_or_else(|| {
-			SubstrateError::ResponseParseFailed(
-				"Parsed an empty list of headers, we should always have at least one.".into(),
-			)
-		})?;
+		const WARNING_MSG: &str = "Parsed an empty list of headers, we should always have at least
```

Comment:
When we spin up the relays they try and get the best headers before the pallet's have been initilaized, which leads to this.

This gets propagated all the way to [`process_future_result()`](https://github.com/paritytech/parity-bridges-common/blob/master/relays/utils/src/lib.rs#L214) which treats it as an error. Ideally this would be a warning, but I don't know how to make that change without making _everything_ a warning (which I don't think is correct).

Reply:
Can we delay the relayers to make sure the pallet is initialized?